lib/tasks/temporary/create_profile_fields.rake

```diff
@@ -0,0 +1,7 @@
+namespace :temporary do
+  desc "Create profile fields based on DEV profile" 
+  task create_profile_fields: :environment do
```

Comment:
Is this something we should also put in the seed file or are we planning to do that later?

Reply:
The seeds files uses a mix right now:

```ruby
  ProfileFields::AddBaseFields.call
  ProfileFields::AddLinkFields.call
  ProfileFields::AddWorkFields.call
  coding_fields_csv = Rails.root.join("lib/data/coding_profile_fields.csv")
  ProfileFields::ImportFromCsv.call(coding_fields_csv)
  ProfileFields::AddBrandingFields.call
```

This was originally done when I wrote the various `ProfileFields::*` service objects to exercise both them and the CSV importer. I can change it in a future PR. The set of generated fields is exactly the same anyway, as `dev_profile_fields.csv` was generated by exporting the `profile_fields` table to CSV after running this seed task.